client/src/utils/messages.js

```diff
@@ -34,4 +34,12 @@ export const filterMessages = defineMessages({
     id: 'filterName',
     defaultMessage: 'Name',
   },
+  pleaseInputYourEmail: {
+    id: 'pleaseInputYourEmail',
+    defaultMessage: 'Please input your E-mail!',
+  },
+  email: {
+    id: 'email',
+    defaultMessage: 'E-mail',
+  },
```

Comment:
With this file, I was thinking we could separate "groups" of messages that are similar - so we can create another export here like this to keep our imports smaller:
```
export const loginMessages = defineMessages({
...
});
```
And then instead of importing `filterMessages`, we can import this `loginMessages`